packages/cmf/src/expressions/getIn.includes.js

```diff
@@ -0,0 +1,11 @@
+import _get from 'lodash/get';
+import Immutable from 'immutable';
+import curry from 'lodash/curry';
+
+function getInIncludes(statePath, { context }, immutablePath, value) {
+	return _get(context.store.getState(), statePath, new Immutable.Map())
+		.getIn(immutablePath.split('.'), new Immutable.List())
+		.includes(value);
+}
+
+export default curry(getInIncludes);
```

Comment:
Maybe use a closure here ? which i find more readable and also the function args after statePath don't seem easy to configure in advance as it is linked with cmf context. Not a issue if we stay like this.